pipeline/vars/common.groovy

```diff
@@ -31,7 +31,7 @@ def getCLIArgsFromMessage() {
         def jsonCIMsg = jsonParser.parseText("${params.CI_MESSAGE}")
 
         env.composeId = jsonCIMsg.compose_id
-        if (! "${env.composeUrl}" ) {
+        if (! env.composeUrl ) {
```

Comment:
Could you use unary operator? i.e. `env.composeUrl = env.composeUrl?.trim() ?: jsonCIMsg.compose_url`